drop-boxes/register-raw-and-mzML/register-raw-and-mzml.py

```diff
@@ -0,0 +1,269 @@
+"""
+This etl script registers a pair consisting of a proteomics raw file or folder and its converted mzML and registers
+both to the same openBIS sample. Some metadata is parsed from the mzML and stored.
+
+Incoming raw files (of different vendor formats) are supported.
+
+This script reuses some logic from the raw file conversion etl script
+
+The stdout of this file is redirected to
+`~openbis/servers/datastore_server/log/startup_log.txt`
+"""
+
+import sys
+sys.path.append('/home-link/qeana10/bin/')
+import os
+import time
+import re
+import subprocess
+import datetime
+import xml.etree.ElementTree
+from functools import partial
+import ch.systemsx.cisd.etlserver.registrator.api.v2
+from ch.systemsx.cisd.openbis.generic.shared.api.v1.dto import (
+    SearchCriteria, SearchSubCriteria
+)
+######## Sample Tracking related import
+from life.qbic.sampletracking import SampleTracker
+from life.qbic.sampletracking import ServiceCredentials
+from java.net import URL
+
+import sample_tracking_helper_qbic as tracking_helper
+#### Setup Sample Tracking service
+SERVICE_CREDENTIALS = ServiceCredentials()
+SERVICE_CREDENTIALS.user = tracking_helper.get_service_user()
+SERVICE_CREDENTIALS.password = tracking_helper.get_service_password()
+SERVICE_REGISTRY_URL = URL(tracking_helper.get_service_reg_url())
+DATA_AVAILABLE_JSON = tracking_helper.get_data_available_status_json()
+
+### We need this object to update the sample status later
+SAMPLE_TRACKER = SampleTracker.createLocationIndependentSampleTracker(SERVICE_REGISTRY_URL, SERVICE_CREDENTIALS)
```

Comment:
Is this necessary? As far as i know we don't show protein based sample type in the sample overview? 
![Bildschirmfoto 2023-11-10 um 10 59 21](https://github.com/qbicsoftware/etl-scripts/assets/29627977/d2f19899-e095-4638-aadf-b0fcfc0914c9)

Reply:
good point, but all of our etl-scripts registering raw data inform the sample tracker, so I'm hesitant to change that.